### PR TITLE
feat(hardhat-deploy-scripts): add missing FPL deploy scripts

### PR DESCRIPTION
**Pull request by abg4**
Signed-off-by: Alex Gaines <againes@umaproject.org>

**Motivation**

The SimpleSuccessToken, SuccessToken, and CappedYieldDollar financial product libraries are missing deploy scripts. This PR includes them.

**Summary**

Adds three deployment scripts. I used the scripts to deploy contracts on Rinkeby. Disregard PR if excluding these was intentional.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/core/deploy/041_deploy_long_short_pair_libraries_simple_success_token.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const { deployer } = await getNamedAccounts();
+
+  await deploy("SimpleSuccessTokenLongShortPairFinancialProductLibrary", {
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+  });
+};
+module.exports = func;
+func.tags = ["LongShortPairLibraries", "lsplib", "SimpleSuccessTokenLongShortPairLibrary"];
```
Comment (mrice32) [inline packages/core/deploy/041_deploy_long_short_pair_libraries_simple_success_token.js:14]: What's the difference between the lsplib tag and the LongShortPairLibraries tag?

**Review by mrice32 — APPROVED** [review]
Looks great!

File: packages/core/deploy/041_deploy_long_short_pair_libraries_simple_success_token.js (hunk 1 above)
Comment (abg4) [reply packages/core/deploy/041_deploy_long_short_pair_libraries_simple_success_token.js:14]: Sorry, I missed this being approved. I am not sure on the reason for both the lsplib and LongShortPairLibraries tag as they are included in the same files so it appears redundant. Do you want me to remove either of them? Also, I will update the file names as it looks like a few new deploy scripts were merged.

**Comment by smb2796** [comment]
@abg4 is this waiting on something, or can it be merged in?

**Pull request merged**